client-spec.md

```diff
@@ -231,7 +231,9 @@ The Verifier now constructs the payload to be signed from the artifact and the a
 
 * Using the raw bytes of the artifact as the payload.
 * Hashing the artifact, then using the resultant digest as the payload.
-* Using [DSSE](https://github.com/secure-systems-lab/dsse/blob/master/protocol.md) as an envelope for the payload with a known DSSE payload type.
+* Using [DSSE](https://github.com/secure-systems-lab/dsse/blob/master/protocol.md) as an envelope for the payload which MUST be an in-toto statement.
+    * Verifier MUST ensure that the artifact's digest/alg tuple is present in the list of subject in the in-toto statement.
```

Comment:
```suggestion
    * Verifier MUST ensure that the artifact's digest/algorithm tuple is present in the list of subject(s) in the in-toto statement.
```

This proposed change is to hint that the in-toto statement might have multiple subjects.